Nos/Models/EventError.swift

```diff
@@ -0,0 +1,28 @@
+import Foundation
+
+/// Errors for an ``Event``.
+enum EventError: Error, LocalizedError {
```

Comment:
`LocalizedError` inherits from `Error` so I think we can get rid of `Error` here.

Reply:
Done!